### ISSUE TITLE
Change the LOGO

### ISSUE THREAD

**Issue by ayush-agarwal-190** [ayush-agarwal-190/Event_Page#26]

**Comment by CDC-Sumantaraj** [comment]
Do i have to give mentors mentee logo ?

**Comment by ayush-agarwal-190** [comment]
Yes we can add that , or for now we can add CDC logo also , its upto you

**Comment by CDC-Sumantaraj** [comment]
can you give me the png file in whatsapp

**Comment by ayush-agarwal-190** [comment]
[Refference2](https://www.google.com/imgres?imgurl=https%3A%2F%2Fcuriousdevelopers.in%2Fwp-content%2Fuploads%2F2022%2F12%2Fcropped-Color-logo-no-background.png&tbnid=tujprH4POGcYLM&vet=12ahUKEwiUpJibg56DAxW-bGwGHb4RA8gQMygAegQIARBP..i&imgrefurl=https%3A%2F%2Fcuriousdevelopers.in%2F&docid=GSQ29Hc_GI9BYM&w=3163&h=776&q=curious%20developers%20community%20CDC%20&ved=2ahUKEwiUpJibg56DAxW-bGwGHb4RA8gQMygAegQIARBP)

[Reff3](https://media.licdn.com/dms/image/C560BAQES4KyHDtjIwQ/company-logo_200_200/0/1672029683686/thecuriousorganization_logo?e=1710979200&v=beta&t=jSdwgSIbrl3_8x51gnJx4wD8KgeVYwCs6butfNDCn5M)



[Reffrence](https://www.google.com/imgres?imgurl=https%3A%2F%2Fcuriousdevelopers.in%2Fwp-content%2Fuploads%2F2022%2F11%2Fhero-img.svg&tbnid=W-5RCTa2gz2-bM&vet=12ahUKEwiUpJibg56DAxW-bGwGHb4RA8gQMygBegQIARBR..i&imgrefurl=https%3A%2F%2Fcuriousdevelopers.in%2F&docid=GSQ29Hc_GI9BYM&w=512&h=444&q=curious%20developers%20community%20CDC%20&ved=2ahUKEwiUpJibg56DAxW-bGwGHb4RA8gQMygBegQIARBR)
